src/terminal.rs

```diff
@@ -239,15 +238,17 @@ impl Terminal {
                 Ok(Key::Char('y')) | Ok(Key::Char('Y')) => break Ok(true),
                 Ok(Key::Char('s')) | Ok(Key::Char('S')) => {
                     println!("\n\nDropping you to shell. Fix what you need and then exit the shell.\n");
-                    run_shell();
+                    run_shell()?;
```

Comment:
Intendet behaviour?

Reply:
Hm, maybe we should print the error instead.

Reply:
Jup, because else it just returns an error, and ignores that we also have error handling later in the function.